### PR TITLE
[torch] fix comment typo

### PR DESCRIPTION
**Pull request by wilson100hong**
Summary: as title

Differential Revision: D49841787

### CODE REVIEW

**Review by Skylion007 — APPROVED** [review]

File: torch/export/_trace.py [hunk 1]
```diff
@@ -506,7 +506,7 @@ def _export(
     operations inside and produce a ExportedProgram.
 
     Args:
-        m: the `nn.Module` or callable to trace.
+        f: the callable to trace.
```
Comment (zhxchen17) [inline torch/export/_trace.py:509]: Actually, this should only support nn.Module, not the other way around

Comment (zhxchen17) [reply torch/export/_trace.py:509]: We can also change the type annotation of `f` to be torch.nn.Module here.

**Review by zhxchen17 — CHANGES_REQUESTED** [review]
the doc needs some update

**Review by zhxchen17 — APPROVED** [review]
thanks!

**Pull request closed without merging**